src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -48,6 +48,9 @@ export const validateEncryptSubmissionParams = celebrate({
       )
       .optional(),
     paymentReceiptEmail: Joi.string(),
+    payments: Joi.object({
+      amount_cents: Joi.number().integer().positive(),
```

Comment:
just curious, do we need to verify the amount limit range?

Reply:
Great catch. I think it makes sense to have config min+max check here. dad1b3ba6